x-pack/plugin/transform/src/main/java/org/elasticsearch/xpack/transform/persistence/IndexBasedTransformConfigManager.java

```diff
@@ -304,13 +283,13 @@ public void deleteOldIndices(ActionListener<Boolean> listener) {
             IndicesOptions.LENIENT_EXPAND_OPEN
         );
 
-        executeAsyncWithOrigin(client, TRANSFORM_ORIGIN, TransportDeleteIndexAction.TYPE, deleteRequest, ActionListener.wrap(response -> {
+        executeAsyncWithOrigin(TransportDeleteIndexAction.TYPE, deleteRequest, listener.delegateFailureAndWrap((l, response) -> {
```

Comment:
Would it be possible to use either `l` or `delegate` parameter name consistently throughout this file?

Reply:
Sure thing moved it all to `l` :)